src/wrapped_radix_engine_toolkit/high_level/ret_api.rs

```diff
@@ -0,0 +1,420 @@
+use crate::prelude::*;
+
+#[uniffi::export]
+pub fn manifest_for_faucet(
+    include_lock_fee_instruction: bool,
+    address_of_receiving_account: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::faucet(
+        include_lock_fee_instruction,
+        address_of_receiving_account,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_marking_account_as_dapp_definition_type(
+    account_address: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::marking_account_as_dapp_definition_type(
+        account_address,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_set_owner_keys_hashes(
+    address_of_account_or_persona: &AddressOfAccountOrPersona,
+    owner_key_hashes: Vec<PublicKeyHash>,
+) -> TransactionManifest {
+    TransactionManifest::set_owner_keys_hashes(
+        address_of_account_or_persona,
+        owner_key_hashes,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_fungible_token_with_metadata(
+    address_of_owner: &AccountAddress,
+    initial_supply: Decimal192,
+    metadata: TokenDefinitionMetadata,
+) -> TransactionManifest {
+    TransactionManifest::create_fungible_token_with_metadata(
+        address_of_owner,
+        initial_supply,
+        metadata,
+    )
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_fungible_token(
+    address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::create_fungible_token(address_of_owner)
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_multiple_fungible_tokens(
+    address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::create_multiple_fungible_tokens(address_of_owner)
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_non_fungible_token(
+    address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    TransactionManifest::create_non_fungible_token(address_of_owner)
+}
+
+#[uniffi::export]
+pub fn manifest_for_create_multiple_non_fungible_tokens(
+    _address_of_owner: &AccountAddress,
+) -> TransactionManifest {
+    todo!()
+}
+
+#[uniffi::export]
+pub fn manifest_stakes_claim(
+    _account_address: &AccountAddress,
+    _stake_claims: Vec<StakeClaim>,
+) -> TransactionManifest {
+    todo!()
+}
+
+#[uniffi::export]
+pub fn manifest_third_party_deposit_update(
+    _to: ThirdPartyDeposits,
+    _owner: &AccountAddress,
+) -> TransactionManifest {
+    todo!()
+}
+
+/// REQUIRES NETWORK CALL (and probable cache)
+/// Requires kotlinx to be setup
+// #[uniffi::export]
+// pub async fn manifest_assets_transfers(
```

Comment:
Such manifest can be constructed with input
```
fromAccount: Network.Account,
depositFungibles: List<FungibleTransfer>,
depositNFTs: List<NonFungibleTransfer>
```
where `FungibleTransfer` and `NonFungibleTransfer` cover these parameters:
```
data class FungibleTransfer(
    val toAccountAddress: String,
    val resourceAddress: String,
    val amount: BigDecimal,
    val signatureRequired: Boolean
)

data class NonFungibleTransfer(
    val toAccountAddress: String,
    val globalId: String,
    val signatureRequired: Boolean
)
```

Reply:
Now we can use strongly typed addresses, instead of raw strings :)